1.8/usage/cli/install.md

```diff
@@ -41,22 +42,62 @@ The easiest way to install the DC/OS CLI is by clicking the signpost icon in the
     ```
 
     You are now ready to use the CLI. Enter `dcos` to get started.
-
-1.  Authenticate your CLI with your master node and set the auth token:
-   
+    
     ```bash
-    $ dcos auth login
```

Comment:
I would keep this and just ask them to follow the instructions printed by the DC/OS CLI after they run the command: `dcos auth login`.